metadata-io/src/main/java/com/linkedin/metadata/search/elasticsearch/indexbuilder/ESIndexBuilder.java

```diff
@@ -861,7 +1183,13 @@ private void diff(String indexA, String indexB, long maxDocs) {
     }
   }
 
-  private long getCount(@Nonnull String indexName) throws IOException {
+  public long getCount(@Nonnull String indexName) throws IOException {
```

Comment:
I didn't see this before I commented above. Is this better than simply setting the refresh_interval equal or slightly less than the polling interface for getCount()?

Reply:
Ultimately, maybe using the task's progress is better if available, defaulting to this if it is missing.